### DISCUSSION TITLE
freeglut install on macos

### DISCUSSION THREAD

**Discussion by Tuychiev99** [community/community#76376] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

help me to install freeglut-3.4.0 on macos in command terminal

**Comment by CheetahCodes21** [comment]
To install freeglut-3.4.0 on macOS using the command terminal, follow these steps:

1. **Install Homebrew:** If you haven't already, install Homebrew, a package manager for macOS. Open the Terminal application and enter the following command:

```bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

2. **Install freeglut:** Once Homebrew is installed, you can install freeglut using the following command:

```bash
brew install freeglut
```

This will download and install freeglut on your macOS system.

3. **Verify the installation:** To verify that freeglut is installed correctly, you can check its version using the following command:

```bash
freeglut --version
```

This should output the installed version of freeglut.
